hack/sync.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# This script sync CRDs from upstream prometheus-operator helm chart
+
+version=${VERSION:-main}
+tmp="/tmp/helm-charts-$version"
```

Comment:
I would prefer something like

```suggestion
tmp="$(mktemp -d -t helm-charts.XXXXX)"
```